RELEASE.md

```diff
@@ -35,17 +35,27 @@ and failures we do not want our users to endure any extra pain.
 
 1. go back to ember-cli directory
 * `git add` the modified `package.json` and `CHANGELOG.md`
-* Commit the changes `git commit -m "Release vx.y.z"` and push `git push`
-* `git tag "vx.y.z"`
-* `git push origin <vx.y.z>`
-* publish to npm
-* `npm publish ./ember-cli-<version>.tgz`
+* if publishing a stable release
+  * Commit the changes `git commit -m "Release vx.y.z"` and push `git push`
+  * `git tag "vx.y.z"`
+  * `git push origin <vx.y.z>`
+  * publish to npm
+  * `npm publish ./ember-cli-<version>.tgz`
+* if publishing a unstable channel release
+  * Commit the changes `git commit -m "Release vx.y.z-channel.n"` and push `git push`
+  * `git tag "vx.y.z-channel.n"`
+  * `git push origin <vx.y.z-channel.n>`
+  * publish to npm
+  * `npm publish ./ember-cli-<version>.tgz --tag x.y.z-channel.n`
```

Comment:
actually, since npm does not have a shrinkwrap file. I am somewhat concerned to release aliases like `beta`, although i think we should, it may be a :footprints: :gun: